Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroGraphics/Driver/MicroGraphics.cs

```diff
@@ -19,7 +19,15 @@ public partial class MicroGraphics
         /// <summary>
         /// PixelBuffer draw target
         /// </summary>
-        protected IPixelBuffer pixelBuffer;
+        protected IPixelBuffer PixelBuffer
+        {
+            get
+            {
+                if (display != null) return display.PixelBuffer;
```

Comment:
By our coding conventions we always add { } even for one line ifs

Reply:
Other option is do:

`return (display != null) ? display.PixelBuffer : _memoryBuffer; `

Reply:
yeah I missed that - I'll fix